libavformat/tls_openssl.c

```diff
@@ -329,8 +330,8 @@ static int openssl_gen_certificate(EVP_PKEY *pkey, X509 **cert, char **fingerpri
         goto enomem_end;
     }
 
-    serial = (int)av_get_random_seed();
-    if (ASN1_INTEGER_set(X509_get_serialNumber(*cert), serial) != 1) {
+    serial = av_get_random_seed();
```

Comment:
serial should be positive integer, see https://github.com/ossrs/ffmpeg-webrtc/pull/51